app/src/main/java/com/example/criminalintent/presentation/crime_detail/CrimeFragment.kt

```diff
@@ -18,7 +19,7 @@ import java.util.*
 
 class CrimeFragment :
     Fragment(),
-    DatePickerFragment.Callbacks {
+    FragmentResultListener {
```

Comment:
`Fragment()` and `FragmentResultListener` can be on the same line as `CrimeFragment`